idunn/utils/categories.yml

```
@@ -70,514 +69,563 @@ categories:
           - "piscines (établissements)"
           - "clubs de sport"
         raw_filters:
-          - "sports_centre,*"
+          - class: "sports_centre"
         regex: "fitness|muscu|piscine"
 
     food_french:
         raw_filters:
-          - "restaurant,french"
+          - class: "restaurant"
+            cuisine: "french"
         pj_what: "restaurant français"
 
     food_pizza:
         raw_filters:
-          - "restaurant,pizza"
+          - class: "restaurant"
+            cuisine: "pizza"
+          - class: "fast_food"
+            cuisine: "pizza"
         pj_what: "pizzeria"
 
     food_burger:
         raw_filters:
-          - "restaurant,burger"
+          - class: "restaurant"
+            cuisine: "burger"
+          - class: "fast_food"
+            cuisine: "pizza"
```

Seems like a small typo:

```suggestion
            cuisine: "burger"
```

---

Thanks! It was certain that I would miss something in this file.